src/SearchParameters/RefinementList.js

```diff
@@ -118,19 +121,19 @@ var lib = {
    * @param {string} refinementValue value of the filter/refinement
    * @return {boolean}
    */
-  isRefined : function isRefined( refinementList, attribute, refinementValue ) {
-    var containsRefinements = refinementList[ attribute ] &&
-                              refinementList[ attribute ].length > 0;
+  isRefined: function isRefined(refinementList, attribute, refinementValue) {
+    var containsRefinements = refinementList[attribute] &&
+      refinementList[attribute].length > 0
 
-    if( isUndefined( refinementValue ) ) {
-      return containsRefinements;
+    if (isUndefined(refinementValue)) {
+      return containsRefinements
     }
 
-    var refinementValueAsString = "" + refinementValue;
+    var refinementValueAsString = '' + refinementValue
 
     return containsRefinements &&
-           refinementList[ attribute ].indexOf( refinementValueAsString ) !== -1;
+      refinementList[attribute].indexOf(refinementValueAsString) !== -1
```

Comment:
Seems not to be enforced by any rule in eslint.

The only added value is that you do not have to hit multiple times <TAB> to align nor you have to precisely space with one more space keypress since it's an even spacing align here (11 spaces).

I think it got rewritten by [standard-format](https://github.com/maxogden/standard-format) when I first used it. standard-format uses esformatter and esformatter has an option for this kind of alignment rule.

I think I will create an esformatter algolia config so that we can easily reformat our code.